content/06-sqlite/06-update-data.mdx

```diff
@@ -154,3 +154,75 @@ SELECT * FROM author;
 ## Conclusion
 
 In this guide, we took a look at the basic ways that you can modify existing data within a table using the `UPDATE` command. Execution of these basic concepts allows you to specify the exact criteria necessary to identify the existing rows within a table, update column names with values, and optionally return the rows that were impacted with `RETURNING`. The `UPDATE` command is critical for managing your data after the initial insertion into your databases.
+
+## FAQ
+
+<details><summary>How do you perform a bulk update in SQLite?</summary>
+
+In SQLite, you can use the `UPDATE` command looking something like the following:
+
+```
+UPDATE my_table
+SET
+        column1 = value1,
+        column2 = value2,
+WHERE
+        id = 1;
+```
+
+This syntax allows you to update in bulk depending on the `WHERE` critera to determine which records match.
+
+</details>
+
+<details><summary>How do you return records modified by the `UPDATE` command in SQLite?</summary>
+
+By default, SQLite does not show the nummber of rows impacted by an `UPDATE` statement.
+
+However, SQLite does have the `RETURNING` clause to return all or part of the records that were modifed.
+
+The basic syntax looks like this with `*` to return all of the columns:
```

Comment:
It's a bit unclear here what the wildcard is standing in for here.  Is it column names?  It might be good to mention that.